tally/CMakeLists.txt

```diff
@@ -14,6 +14,8 @@ file(GLOB PUB_HEADERS ${PUB_HEADERS} "*.hpp")
 # Libraries
 set(LINK_LIBS)
 list(APPEND LINK_LIBS ${DAG_LINK_LIBS})
+list(APPEND LINK_LIBS ${GFORTRAN_LIBRARY})
+list(APPEND LINK_LIBS ${QUADMATH_LIBRARY})
```

Comment:
This may be a dumb question, but what are libgfortran and libquadmath used for? When I removed the lapack dependency in #462, I was also able to remove the quadmath and gfortran dependencies as well.